test_collections/sdk_tests/support/python_testing/models/utils.py

```diff
@@ -76,13 +50,8 @@ def generate_command_arguments(
     # Retrieve arguments from test_parameters
     if test_parameters:
         for name, value in test_parameters.items():
-            if name in valid_args:
-                if str(value) != "":
-                    arguments.append(f"--{name.replace('_','-')} {str(value)}")
-                else:
-                    arguments.append(f"--{name.replace('_','-')} " "")
-            else:
-                logger.warning(f"Argument {name} is not valid")
+            arg_value = str(value) if str(value) != "" else " "
```

Comment:
Is this `" "` space needed? Why not just let the empty string?

Reply:
I agree with you, I was keeping that way it was before, but I think it not required to be `" "`